### ISSUE TITLE
Specifying fontSize for normal text is not possible

### ISSUE THREAD

**Issue by sharnik** [mientjan/react-native-markdown-renderer#17]
I'm trying to set a different `fontSize` for the general text and for headings. However, when I set the `fontSize` style for `text` it'll overwrite any `fontSize` styles for `heading` or `headingX`.
My guess is: it's because the text node gets nested inside the heading, so overwrites the style.

Is there a workaround for that? Is there a way to add style to just normal paragraph text? (I can't use `paragraph`, because it's for styling a `View`). 

Is there anything I could help with to add this feature?

**Comment by jonsamp** [comment]
Same issue. Also, when I add styles to `text`, it causes all inline code to have a line break.

Here's my code:
```js
<Markdown style={{ text: { fontSize: 18 } }} >Here is `code` with some more `code`!</Markdown>
```

I would expect it to render text like:
Here is `code` with some more `code`!

But instead it renders as:
Here is 
`code` with some more 
`code`!

**Comment by mientjan** [comment]
This is now fixed in release/3.0.0. I'm still testing stuff out but will be pushed in a day or 2. The problem was that text was always the leaf node that was rendered. So it would always have the last say about styling.

Now you have textgroup renderRule that wraps all text nodes. So when a sub node is not applying a style like fontSize that node will inherit it from textgroup.

**Comment by fiznool** [comment]
I'm still having trouble with this. Could you clarify how to change the text size for just paragraph text, leaving the headings, list items etc alone?
